web/index.html

```diff
@@ -4,8 +4,9 @@
     <meta charset="UTF-8">
     <title>IPFS Check</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" href="tachyons.min.css"/>
+    <link rel="stylesheet" href="/tachyons.min.css"/>
     <link rel="canonical" href="https://ipfs-check.on.fleek.co/">
+    <script type="text/javascript" src="/dist/telemetry.js" ></script>

```

Comment:
Forgive my ignorance here, but why do these have `/` in front now and require running npm?

Any reason things can't be left simply where you can just open up `web/index.html` and everything just works?

Reply:
The / is not related to the build, just, more idiomatic for representing relative asset.

The reason things can't be "just open up web/index.html and everything works" is because our ignite-metrics package isn't fully exporting browser compatible esm imports at the moment, so we require a build step.

We could commit a minified telemetry.min.js file and include that, but i'd rather we didn't.

Reply:
> The / is not related to the build, just, more idiomatic for representing relative asset.

You might know better than me here, but this idiom doesn't really work if you just load index.html locally. Instead everything breaks as it tries to load from `/` on your machine using `file://` (at least on Chrome + Windows).

When I remove the `/` and load everything seems to work fine, and I don't have any console errors. Perhaps the telemetry isn't reporting properly then but idk.